plugin/src/main/groovy/com/btkelly/gnag/tasks/GnagCheck.java

```diff
@@ -90,7 +95,7 @@ private void executeGnagCheck() {
                                 violationDetector.name() + " detected " + detectedViolations.size() + " violations.");
                 });
 
-        final File reportsDir = reportHelper.getReportsDir();
+        final File reportsDir = projectHelper.getReportsDir();
```

Comment:
I thought about that, maybe just renaming `ProjectHelper` again. I didn't want to create a ton of util type classes just because the name didn't quite fit.